RCSwitch.cpp

```diff
@@ -259,6 +259,26 @@ void RCSwitch::switchOff(const char* sGroup, const char* sDevice) {
 }
 
 /**
+ * Switch a remote switch on (Type E RSL by Conrad)
+ *
+ * @param sGroup   Number of the switch group (1..4)
+ * @param nDevice  Number of the switch itself (1..4)
+ */
+void RCSwitch::switchOnRSL(int sGroup, int nDevice) {
+	this->send( this->getCodeWordE(sGroup, nDevice, true) );
+}
+
+/**
+ * Switch a remote switch off (Type E RSL by Conrad)
+ *
+ * @param sGroup   Number of the switch group (1..4)
+ * @param nDevice  Number of the switch itself (1..4)
+ */
+void RCSwitch::switchOffRSL(int sGroup, int nDevice) {
+	this->send( this->getCodeWordE(sGroup, nDevice, false) );
```

Comment:
Could you please use two spaces for indention, instead of tabs? To match the surrounding code... :-)